.github/workflows/mutation-tests.yml

```diff
@@ -65,7 +65,9 @@ jobs:
           vendor/bin/roave-infection-static-analysis-plugin \
             --ignore-msi-with-no-mutations \
             --git-diff-filter=AM \
-            --git-diff-base=origin/${{ github.base_ref }}
+            --git-diff-base=origin/${{ github.base_ref }} \
+            --min-covered-msi=100 \
```

Comment:
These can/should be set in `infection.json.dist` (that way, you get same results also locally)